spk-config.yaml

```diff
@@ -1,11 +1,11 @@
 # Azure DevOps Configuration
 azure_devops:
-  access_token: "hpe3a9oiswgcodtfdpzfiek3saxbrh5if1fp673xihgc5ap467a" # This is your Personal Access Token with permission to modify and access this private repo. Leave this empty if project is public
-  hld_repository: "https://dev.azure.com/bhnook/fabrikam/_git/hld" # Repository URL for your Bedrock HLDs
-  manifest_repository: "https://dev.azure.com/bhnook/fabrikam/_git/materialized" # Repository URL that is configured for flux. This holds the kubernetes manifests that is generated by fabrikate.
-  infra_repository: "https://dev.azure.com/naros32/bedrock_testing_phase2/_git/spk-testing-private-azdo" # Repository URL that contains your terraform templates to be sed for scaffolding and generating infrastructure deployment templates.
-  org: "bhnook" # Your AzDo Org
-  project: "fabrikam" # Your AzDo project
+  access_token: "" # This is a Personal Access Token with permission to modify and access the HLD, manifest and infra repos. Leave this empty if project is public
+  hld_repository: "" # Repository URL for your Bedrock HLDs
+  manifest_repository: "" # Repository URL for yor materialized manifests generated by fabrikate.
```

Comment:
your

Reply:
The URLs with real GitHub users that could be interpreted as repos that should already exist were confusing. Generic URLs that show the pattern but make it clear to the user that those are project specific could be helpful.

Reply:
added dummy URLs, fixed type